phpstan.neon

```diff
@@ -3,7 +3,7 @@ includes:
 parameters:
     paths:
         - ./
-    level: 1
+    level: 2
```

Comment:
can you reset this file? then we could merge the fixes that are already implemented 🙂